Framework/Crystal/src/IndexPeaks.cpp

```diff
@@ -418,9 +418,10 @@ std::map<std::string, std::string> IndexPeaks::validateInputs() {
     ws->sample().getOrientedLattice();
   } catch (std::runtime_error &exc) {
     helpMsgs[Prop::PEAKSWORKSPACE] = exc.what();
+    return helpMsgs;
```

Comment:
Can I check that the early return is because it would crash if left to continue to the end?

It works as expected but the help message is more developer friendly: `Sample::getOrientedLattice() - No OrientedLattice has been defined`. It may be clearer to say something like "No UB matrix defined".

Reply:
You're right, the early return is needed because the `parse` method now used in this function makes use of the lattice, and throws an unhandled exception if not present. Will update the message